test/framework/.vscode/launch.json

```diff
@@ -0,0 +1,30 @@
+{

```

Comment:
is this file meant to be included?  If it is, please can it have comments in english?

Reply:
IDE files should never be included. Maybe add to `.gitignore`?

Reply:
You are right

Reply:
Deleted and added .gitignore